projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -94,18 +94,31 @@ module.exports = async function (...args) {
 		buildSoy();
 	}
 
-	runBabel(
-		BUILD_CONFIG.input,
-		'--out-dir',
-		BUILD_CONFIG.output,
-		'--source-maps'
-	);
+	const disableOldBuild =
+		FEDERATION_CONFIG && FEDERATION_CONFIG.disableOldBuild;
+
+	if (!disableOldBuild) {
+		runBabel(
+			BUILD_CONFIG.input,
+			'--out-dir',
+			BUILD_CONFIG.output,
+			'--source-maps'
+		);
+	}
 
-	if (fs.existsSync('webpack.config.js') || !!FEDERATION_ENABLED) {
+	if (fs.existsSync('webpack.config.js') || !!FEDERATION_CONFIG) {
 		webpack(...args);
 	}
 
-	runBundler();
+	if (!disableOldBuild) {
+		runBundler();
+	}
+	else {
+		const {output} = BUILD_CONFIG;
```

Comment:
These steps are usually performed by the bundler so we need to "fake" them here.